lib/twine/formatters/android.rb

```diff
@@ -45,7 +45,7 @@ def output_path_for_language(lang)
       end
 
       def set_translation_for_key(key, lang, value)
-        value = CGI.unescapeHTML(value)
+        value = CGI.unescapeHTML((value.nil? || value.empty?) ? "" : value)
```

Comment:
Yes, parsing will throw an error (and cause no more strings to be processed for that file), in 2 cases:

1) String with no values ex. `<string name="device.troubleshoot"></string>`
2) Strings whose value is a single HTML element ex. `<string name="device.skip"><u>Skip</u></string>`